app/controllers/v0/sign_in_controller.rb

```diff
@@ -103,10 +103,11 @@ def token
 
       StatsD.increment(SignIn::Constants::Statsd::STATSD_SIS_TOKEN_SUCCESS)
       render json: response_body, status: :ok
-    rescue SignIn::Errors::StandardError => e
+    rescue SignIn::Errors::StandardError, Login::Errors::CredentialLockedError => e
+      status = e.is_a?(Login::Errors::CredentialLockedError) ? :forbidden : :unauthorized
       sign_in_logger.info('token error', { errors: e.message })
       StatsD.increment(SignIn::Constants::Statsd::STATSD_SIS_TOKEN_FAILURE)
-      render json: { errors: e }, status: :bad_request
+      render json: { errors: e }, status:
```

Comment:
Same here, since it's a bit on the fence whether to use `forbidden` or `bad_request`, I'd prob just use `bad_request` for simplicity sake